### PR TITLE
Yarn :: NodeJS 12.18.3 (LTS) and 14.10.0 (latest)

### PR DESCRIPTION
**Pull request by eeertekin**

### CODE REVIEW

File: yarn/cloudbuild.yaml [hunk 1]
```diff
@@ -63,3 +78,5 @@ images:
 - 'gcr.io/$PROJECT_ID/yarn:node-8.4.0'
```
Comment (bendory) [inline yarn/cloudbuild.yaml:78]: Please add `:lts` to the list of pushed images.

**Review by bendory — CHANGES_REQUESTED** [review]
LGTM once `:lts` is added.

File: yarn/cloudbuild.yaml (hunk 1 above)
Comment (eeertekin) [reply yarn/cloudbuild.yaml:79]: @bendory :lts tag is added to image list

Thank you!

**Comment by eeertekin** [comment]
Hey @bendory,

I'm still getting error on Cloud Build when I try to use gcr.io/cloud-builders/yarn:14.10.0

I tried it on my own docker, it is working fine but Cloud Build is not able to reach the new images. 

Is there anything I can help about?

Thanks!

`Pulling image: gcr.io/cloud-builders/yarn:14.10.0`
`Error response from daemon: manifest for gcr.io/cloud-builders/yarn:14.10.0 not found: manifest unknown: Failed to fetch "14.10.0" from request "/v2/cloud-builders/yarn/manifests/14.10.0".`
`Error response from daemon: manifest for gcr.io/cloud-builders/yarn:14.10.0 not found: manifest unknown: Failed to fetch "14.10.0" from request "/v2/cloud-builders/yarn/manifests/14.10.0".`
`Error response from daemon: manifest for gcr.io/cloud-builders/yarn:14.10.0 not found: manifest unknown: Failed to fetch "14.10.0" from request "/v2/cloud-builders/yarn/manifests/14.10.0".`
`Error response from daemon: manifest for gcr.io/cloud-builders/yarn:14.10.0 not found: manifest unknown: Failed to fetch "14.10.0" from request "/v2/cloud-builders/yarn/manifests/14.10.0".`
`Error response from daemon: manifest for gcr.io/cloud-builders/yarn:14.10.0 not found: manifest unknown: Failed to fetch "14.10.0" from request "/v2/cloud-builders/yarn/manifests/14.10.0".`
`Error response from daemon: manifest for gcr.io/cloud-builders/yarn:14.10.0 not found: manifest unknown: Failed to fetch "14.10.0" from request "/v2/cloud-builders/yarn/manifests/14.10.0".`
`Error response from daemon: manifest for gcr.io/cloud-builders/yarn:14.10.0 not found: manifest unknown: Failed to fetch "14.10.0" from request "/v2/cloud-builders/yarn/manifests/14.10.0".`
`Error response from daemon: manifest for gcr.io/cloud-builders/yarn:14.10.0 not found: manifest unknown: Failed to fetch "14.10.0" from request "/v2/cloud-builders/yarn/manifests/14.10.0".`

**Comment by eeertekin** [comment]
Hey @bendory,

`gcr.io/cloud-builders/yarn:latest` shows that it is on v14.10.0, but when I try to use it on Google Cloud Build,
it is still working with node `v8.12.0`.

Probably related with GCB's worker instances and their image caches, but is there any chance to sync it with latest GCR image on GCB?

Thank you!

**Comment by bendory** [comment]
Merging the PR here does not make the image available in GCB; it needs to go through our CI and release process first. In the interim, you can build the image and use it from your own project.

**Comment by bendory** [comment]
Note that you can always go to http://gcr.io/cloud-builders/yarn to see what versions have been built. The tagged `:node-14:10.0` version is available now.

**Pull request merged**